app/scripts/models/account.js

```diff
@@ -334,7 +337,8 @@ define(function (require, exports, module) {
         if (password) {
           return self._fxaClient.signIn(email, password, relier, {
             metricsContext: self._metrics.getActivityEventMetadata(),
-            reason: options.reason
+            reason: options.reason || SignInReasons.SIGN_IN,
+            resume: options.resume
```

Comment:
After looking at the diff once I added it, I realized yes, it is intentionally left out. [fxa-client.js->signIn](https://github.com/mozilla/fxa-content-server/pull/3671/files#diff-341971be75d35dbd724949cda527a0efR198) fetches the resume token from the relier directly, it does not look at the options.